### ISSUE TITLE
Shouldn't be able to modify project tasks on closed projects

### ISSUE THREAD

**Issue by rdoursenaud** [GPCsolutions/dolibarr#67]

**Comment by marcosgdf** [comment]
Should this be extended to creation and deletion?

Which version do you want to correct?

**Comment by rdoursenaud** [comment]
Yes, IMHO creation and deletion are just special cases of modifications ;)

All our issues target the current stable release but needs inclusion upstream following their rules.

My take on this is usually to use upstream rules as a starting point:
- new feature / enhancement -> work on develop
- bug/misbehavior -> work on earliest affected — still supported — version

Then I (back)port the changes to the current stable for inclusion in our support branch.

But you can also work the other way around. Your call.
